### ISSUE TITLE
CSRF Cookie not set

### ISSUE THREAD

**Issue by ccurvey** [OmniDB/OmniDB#937]
I just tried installing 2.15 via

    $ sudo dpkg -i omnidb-server_2.15.0-debian-amd64.deb

`syslog` tells me that things seemed to start OK.   In `/etc/omnidb.conf` I changed

```
listening_address    = 0.0.0.0                                                                   
listening_port       = 8002         
```

When I bring up the login page in a web browser, I get "request error" in the browser, and this message in syslog

    Apr 30 13:48:55 ip-172-31-21-62 bash[22160]: [04/30/2019 13:48:55] WARNING [django.security.csrf:228] Forbidden (CSRF cookie not set.): /check_session_message/

**Comment by rafaelthca** [comment]
Hi @ccurvey, did you configure `omnidb.conf` with `is_ssl` set to true?

How are you accessing with the browser? Is that a direct access or through a reverse proxy?

Rafael.

**Comment by ccurvey** [comment]
yes, I was trying to set up SSL for my OmniDB server, so I had set `is_ssl=True`.  But I have changed that back to False to try to get anything working again. 

I'm directly hitting `http://fqdn.of.my.server:8002`  No proxy servers are involved (that I know of!)

I thought this might be a cacheing issue, as I'm getting the error in chrome, but not Firefox. I think that's wrong, as clearing Chrome's cache does not seem to change anything.
